Components/Badge/badge.css

```diff
@@ -0,0 +1,187 @@
+.icon-badge-container
+{
+    width: 100%;
+    height: fit-content;
+    box-shadow: rgba(0, 0, 0, 0.02) 0px 1px 3px 0px, rgba(27, 31, 35, 0.15) 0px 0px 0px 1px;
+    display: flex;
+    justify-content: space-evenly;
+    align-items: center;
+    flex-wrap: wrap;
+    padding: 3rem 1rem;
+}
+
+.icon-count-badge
+{
+    display: flex;
+    align-items:center;
+    position: relative;
+}
+
+.count-badge-5x
+{
+    position: absolute;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    background-color:  var(--notification-or-error);
+    border-radius: 100%;
+    color: white;
```

Comment:
- Is Border radius 100% needed (50% should suffice)? 
- avoid using pure white/black and writing color name

Reply:
Done, Thanks for review.